feat: Initial reports (#2)
turbot/steampipe-mod-gitlab-insights: dashboards/group/docs/group_report_expiring_member.md

This report answers the following questions:

- Do my groups have any members with expiring access?

> - When do members with expiring access lose it?

Reviewer: ```suggestion
- When does the access expire for the members?
```
